src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1791,15 +1794,15 @@ void esbmc_parseoptionst::help()
        " --i386-linux                 set Linux/I386 architecture\n"
        " --i386-win32                 set Windows/I386 architecture (default)\n"
 #elif __APPLE__
-       " --i386-macos                 set MACOS/I386 architecture (default)\n"
-       " --ppc-macos                  set PPC/I386 architecture\n"
-       " --i386-linux                 set Linux/I386 architecture\n"
-       " --i386-win32                 set Windows/I386 architecture\n"
+         " --i386-macos                 set MACOS/I386 architecture (default)\n"
```

Comment:
why did you add these spaces?

Reply:
It was a mistake, not sure how it happened though